test-clients/src/main/java/com/hedera/services/bdd/suites/perf/file/MixedFileOpsLoadTest.java

```diff
@@ -0,0 +1,115 @@
+package com.hedera.services.bdd.suites.perf.file;
+
+/*-
+ * ‌
+ * Hedera Services Test Clients
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import com.hedera.services.bdd.spec.HapiApiSpec;
+import com.hedera.services.bdd.spec.HapiSpecOperation;
+import com.hedera.services.bdd.spec.transactions.TxnUtils;
+import com.hedera.services.bdd.spec.utilops.LoadTest;
+import com.hedera.services.bdd.suites.perf.PerfTestLoadSettings;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.function.Supplier;
+
+import static com.hedera.services.bdd.spec.HapiApiSpec.defaultHapiSpec;
+import static com.hedera.services.bdd.spec.queries.QueryVerbs.getFileInfo;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.fileAppend;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.fileCreate;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.fileUpdate;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.logIt;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.withOpContext;
+
+/**
+ * Run mixed operations including FileCreate, FileAppend, FileUpdate
+ */
+public class MixedFileOpsLoadTest extends LoadTest {
+	private static final Logger log = LogManager.getLogger(MixedFileOpsLoadTest.class);
+
+	public static void main(String... args) {
+		parseArgs(args);
+
+		MixedFileOpsLoadTest suite = new MixedFileOpsLoadTest();
+		suite.setReportStats(true);
+		suite.runSuiteSync();
+	}
+
+	@Override
+	protected List<HapiApiSpec> getSpecsInSuite() {
+		return List.of(
+				runMixedFileOps()
+		);
+	}
+
+	@Override
+	public boolean hasInterestingStats() {
+		return true;
+	}
+
+	protected HapiApiSpec runMixedFileOps() {
+		PerfTestLoadSettings settings = new PerfTestLoadSettings();
+		final AtomicInteger submittedSoFar = new AtomicInteger(0);
+		final byte[] newContents = TxnUtils.randomUtf8Bytes(TxnUtils.BYTES_4K);
+		String initialContent = "The initial contents!";
+		String targetFile = "targetFile";
+
+		Supplier<HapiSpecOperation[]> mixedFileOpsBurst = () -> new HapiSpecOperation[] {
+				fileCreate(targetFile + submittedSoFar.getAndIncrement()).contents(initialContent),
+				fileUpdate(targetFile)
+						.fee(ONE_HUNDRED_HBARS)
+						.contents(newContents)
```

Comment:
nit: even though `newContents` are random bytes, they don't change over the course of this load test. If your intention was to have them change, `TxnUtils.randomUtf8Bytes(TxnUtils.BYTES_4K)` must be moved here.

Reply:
Fixed. Thanks @qnswirlds !